skpro/classification/sklearn_classifiers/_sklearn_proba_class.py

```diff
@@ -0,0 +1,251 @@
+"""Adapter to sklearn multiclass classification using Histograms."""
+
+__author__ = ["ShreeshaM07"]
+
+import numpy as np
+import pandas as pd
+
+from skpro.distributions import Histogram
+from skpro.regression.base import BaseProbaRegressor
+from skpro.utils.sklearn import prep_skl_df
+
+
+class MulticlassSklearnProbaClassifier(BaseProbaRegressor):
+    """A multiclass classifier fitting a histogram distribution.
```

Comment:
as said above, the resulting estimator is a probabilistic regressor, not a multiclass classifier. It is a probabilistic regressor that fits a histogram distribution by presenting binned outcomes to a probabilistic classifier from sklearn.

Now I don't know how to make this into a short description the best way, I'm just saying the above needs to be adapted.